schedule/functional/sle16/system_performance.yaml

```diff
@@ -0,0 +1,34 @@
+name:           system_performance
+description:    >
+    Maintainer: qe-core.
+schedule:
+    - installation/bootloader_start
+    - boot/boot_to_desktop
```

Comment:
IMO this is different than 15-SP7 test, because it uses some specific installation setup
https://openqa.suse.de/tests/17639693
Honestly IDK what exactly makes the test system_performance, but here it looks like any other tests which lost the purpose

Reply:
@dzedro Thanks, IMO we can move all of these tests to extra_tests_textmode, [but please make your own decisioin :)]

Reply:
I wouldn't like to move all  of these tests to extra_tests_textmode. As system_performance job has 29  test modules. If every test modules add to extra_tests_textmode, it will make extra_tests_textmode job longer and longer. I thinks separate these tests to different jobs can be easy to review and debug failed test moules.